### DISCUSSION TITLE
npx 快速生成move 项目

### DISCUSSION THREAD

**Discussion by v1xingyue** [NonceGeek/Move-Camp#252] [Materials]
npm 项目地址: 

https://www.npmjs.com/package/create-move-app

使用方法: 

```shell
npx create-move-app [name] --chain=aptos
```

目前支持的链包括 aptos,sui,starcoin 

使用预览:  

<img width="1679" alt="image" src="https://user-images.githubusercontent.com/974169/214812274-207e5cda-9ca5-460b-b995-293a90eae107.png">

**Comment by leeduckgo** [comment]
wow，这个真的好！
